tests/test_compressors/test_model_compressor.py

```diff
@@ -16,11 +16,24 @@
 
 import pytest
 from compressed_tensors.compressors.model_compressor import ModelCompressor
+from compressed_tensors.config.base import SparsityCompressionConfig
+from compressed_tensors.quantization.quant_config import QuantizationConfig
+
+
+def compressed_tensors_config_available():
```

Comment:
could this be combined with `is_compressed_tensors_config`?

Reply:
No. I need a function which returns True if CompressTensorsConfig is available.

I cannot use `is_compressed_tensors_config` since this function only returns true if the passed argument is of type CompressTensorsConfig, and I cannot instantiate an argument of type CompressTensorsConfig if I don't know if CompressTensorsConfig is available, which is circular logic.

Reply:
`is_compressed_tensors_config` could conceivably use `compressed_tensors_config_available` but it would just be more lines in the implementation and wouldn't really add anything.

Reply:
We should consider adding a specific `skipif` marker for CTC, but that can also be a separate PR

Reply:
Moved to `testing_utils.py`